lib/recommendable/rater/recommender.rb

```diff
@@ -8,8 +8,10 @@ module Recommender
       # @return [Array] An array of instances of your user class
       def similar_raters(limit = 10, offset = 0)
         ids = Recommendable.redis.zrevrange(Recommendable::Helpers::RedisKeyMapper.similarity_set_for(id), 0, -1)
+        # Sanitize ids using klass type mapping
+        ids.map!{ |id| self.class.new(id: id).id }.compact
```

Comment:
Probably You should use `Recommendable.config.user_class.primary_key` to stay consistent with query.